[Help Doc] Added help file for instrument data entry (#6902)
aces/Loris: modules/instruments/help/instruments.md

# Instruments

Inside the instrument form, general information is shown at the top including date of birth, sex, visit label, and subproject(cohort).

In the left sidebar panel (on tablets, click the list icon) - you can set status flags for this instrument form.  The Data Entry flag can only be set to Complete after the Administration and Validity flags have been set. 
The Data Entry flag can only be set to Complete after the Administration and Validity flags have been set.
While the Data Entry status is `In Progress`, you may see in the sidebar the option to delete all the instrument data which will clear all data in the form. 

> Under the general information about the candidate at the top of the screen, the user can perform data entry for the instrument. Required values are specified with a red asterisk. If these values are not completed, an error message will appear when a user attempts to save the data. 
> 
> If the instrument has multiple pages, the user must save the data individually for each page. If the user has navigated to a page that is not the `Top` page of the instrument, they will be unable to alter any status flags in the sidebar. 

Reviewer: ```suggestion
```
removing

# Instruments

Inside the instrument form, general information is shown at the top including date of birth, sex, visit label, and subproject(cohort).

> In the left sidebar panel (on tablets, click the list icon) - you can set status flags for this instrument form.  The Data Entry flag can only be set to Complete after the Administration and Validity flags have been set. 

Reviewer: ```suggestion
Status flags can be set in the left sidebar - the Data Entry flag can only be set to Complete after the Administration and Validity flags have been set. 
```

> If a form has multiple pages, enter and save data on each subpage before returning to the Top page to set the status flags. 
> 
> In the sidebar, the Data Entry status can be set to either `Complete` or `In Progress`. The status cannot be set to `Complete` if the Administration option is not set. If the Data Entry status is `In Progress`, the sidebar will display the option to delete all the instrument data. The sidebar also allows the user to specify the Administration status of the instrument. 

Reviewer: ```suggestion
The Data Entry flag can only be set to Complete after the Administration and Validity flags have been set.
While the Data Entry status is `In Progress`, you may see in the sidebar the option to delete all the instrument data which will clear all data in the form. 
```

If a form has multiple pages, enter and save data on each subpage before returning to the Top page to set the status flags. 

In the sidebar, the Data Entry status can be set to either `Complete` or `In Progress`. The status cannot be set to `Complete` if the Administration option is not set. If the Data Entry status is `In Progress`, the sidebar will display the option to delete all the instrument data. The sidebar also allows the user to specify the Administration status of the instrument. 

Under the general information about the candidate at the top of the screen, the user can perform data entry for the instrument. Required values are specified with a red asterisk. If these values are not completed, an error message will appear when a user attempts to save the data. 

If the instrument has multiple pages, the user must save the data individually for each page. If the user has navigated to a page that is not the `Top` page of the instrument, they will be unable to alter any status flags in the sidebar. 

> 
> ## Medical History
> 
> This instrument contains 4 pages of data entry. The required information is on the Top page and is marked with a red asterisk. 
> 
> If the user navigates to a page other than the `Top` page, the Administration and Data Entry options in the sidebar cannot be edited.
> 
> ## BMI Calculator
> 
> The BMI Calculator instrument has one page of data entry. The required information is marked with a red asterisk.
> 
> Along with the Administration and Data Entry options, the sidebar for this instrument includes the option to specify the Validity of the instrument. If the Data Entry status is set to `Complete`, the Validity option cannot be edited.
> 
> ## Radiology Review
> 
> The Radiology Review instrument has one page of data entry. The required information is marked with a red asterisk.
> 
> ## AOSI
> 
> The AOSI instrument has three pages of data entry. The required information is marked with a red asterisk.
> 
> Along with the Administration and Data Entry options, the sidebar for this instrument includes the option to specify the Validity of the instrument. If the Data Entry status is set to `Complete`, the Validity option cannot be edited. If the Adminisitration or Validity options are not set by the user, the Data Entry status cannot be changed to `Complete`.

Reviewer: ```suggestion
```

> # Instrument
> 
> 

Reviewer: ```suggestion
# Instruments

```

# Instruments

Inside the instrument form, general information is shown at the top including date of birth, sex, visit label, and subproject(cohort).

> 
> 

Reviewer: ```suggestion

```
remove extra space
